InstructorCourseJoinConfirmationPageUiTest: Fix broken Javadoc link

Change
```java
/**
 * SUT: {@link Const.ActionURIs#INSTRUCTOR_COURSE_JOIN_CONFIRMATION_PAGE}.
 */
```
to
```java
/**
 * SUT: {@link Const.ActionURIs#INSTRUCTOR_COURSE_JOIN}.
 */
```

The only thing I changed in the branch was the javadoc link. However, I get a failed compileTestJava in after running ./gradlew lint. 

>> ./gradlew lint
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:checkstyleMain UP-TO-DATE
:findbugsMain UP-TO-DATE
:pmdMain UP-TO-DATE
:lintMain UP-TO-DATE
:compileTestJava
/Users/vanderlaan/Desktop/Projects/teammates/src/test/java/teammates/test/pageobjects/AppPage.java:176: error: no suitable method found for getNewPageInstance(Browser,TestProper[...]class)
        return getNewPageInstance(browser, TestProperties.isDevServer() ? DevServerLoginPage.class
               ^
    method AppPage.<T#1>getNewPageInstance(Browser,Url,Class<T#1>) is not applicable
      (cannot infer type-variable(s) T#1
        (actual and formal argument lists differ in length))
    method AppPage.<T#2>getNewPageInstance(Browser,Class<T#2>) is not applicable
      (inferred type does not conform to equality constraint(s)
        inferred: GoogleLoginPage
        equality constraints(s): GoogleLoginPage,DevServerLoginPage)
  where T#1,T#2 are type-variables:
    T#1 extends AppPage declared in method <T#1>getNewPageInstance(Browser,Url,Class<T#1>)
    T#2 extends AppPage declared in method <T#2>getNewPageInstance(Browser,Class<T#2>)
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
1 error
:compileTestJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED.

--- a/src/test/java/teammates/test/cases/browsertests/InstructorCourseJoinConfirmationPageUiTest.java
+++ b/src/test/java/teammates/test/cases/browsertests/InstructorCourseJoinConfirmationPageUiTest.java
@@ -12,7 +12,7 @@ import teammates.test.pageobjects.InstructorCourseJoinConfirmationPage;
 import teammates.test.pageobjects.InstructorHomePage;
 
 /**
- * SUT: {@link Const.ActionURIs#INSTRUCTOR_COURSE_JOIN_CONFIRMATION_PAGE}.
+ * SUT: {@link Const.ActionURIs#INSTRUCTOR_COURSE_JOIN}.
  */
 public class InstructorCourseJoinConfirmationPageUiTest extends BaseUiTestCase {
 

